.env.example

```diff
@@ -1,64 +1,48 @@
-# A Github token to download the CI artifacts
-# Used for fetching resources in the kakarot_scripts
+# Tokens for coverage and CI
+CODECOV_TOKEN=
 GITHUB_TOKEN=
-
-# An infura RPC key for mainnet/testnet
+# API keys
 INFURA_KEY=
+VOYAGER_API_KEY=
+NETHERMIND_API_KEY=
 
-# Account used by default in all kakarot_scripts
-# Prefixed accounts {NETWORK}_ will take precedeance over the default one
-MADARA_ACCOUNT_ADDRESS=0x3
-MADARA_PRIVATE_KEY=0x00c1cf1490de1352865301bb8705143f3ef938f97fdf892f1090dcb5ac7bcd1d
-
-SHARINGAN_ACCOUNT_ADDRESS=
-SHARINGAN_PRIVATE_KEY=
-SHARINGAN_RPC_URL="https://sharingan.madara.zone"
-
-# First account when using these networks with seed = 0
-STARKNET_DEVNET_ACCOUNT_ADDRESS=0x64b48806902a367c8598f4f95c305e8c1a1acba5f082d294a43793113115691
-STARKNET_DEVNET_PRIVATE_KEY=0x71d7bb07b9a64f6f78ac4c816aff4da9
-
-STARKNET_SEPOLIA_RELAYER_ACCOUNT_ADDRESS=
-STARKNET_SEPOLIA_RELAYER_PRIVATE_KEY=
-STARKNET_SEPOLIA_ACCOUNT_ADDRESS=
-STARKNET_SEPOLIA_PRIVATE_KEY=
+# Starknet accounts
+STARKNET_DEVNET_ACCOUNT_ADDRESS=
+STARKNET_DEVNET_PRIVATE_KEY=
+# This default value is Anvil first account private key
+STARKNET_DEVNET_EVM_PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
 
 KATANA_ACCOUNT_ADDRESS=0xb3ff441a68610b30fd5e2abbf3a1548eb6ba6f3559f2862bf2dc757e5828ca
 KATANA_PRIVATE_KEY=0x2bbf4f9fd0bbb2e60b0316c1fe0b76cf7a4d0198bd493ced9b8df2a3a24d68a
+# This default value is Anvil first account private key
```

Comment:
```suggestion
# This default value is Katana first account private key
```

Reply:
no it's anvil first account private key (EVM account)